src/licensedcode/data/licenses/Scilab-old.yml

```diff
@@ -0,0 +1,8 @@
+key: Scilab-old
```

Comment:
The keys and filename are always all lowercase, also is there anything but this old scilab licenses that is a scilab license? If not let's leave aside the old suffix

Reply:
Ok , I think i should rename both key and filename as <code>scilab</code>

Reply:
yes